File: Java/viral4.java
//sum of three number

import java.util.Scanner;

public class viral4 {
    public static void main(String[] args) 
    {
        System.out.println();
        float sum=0;
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the first number :");
        float x = input.nextFloat();
        sum += x;
        System.out.print("Enter the second number :");
        float y = input.nextFloat();
        sum += y;
        System.out.print("Enter the third number :");
        float z = input.nextFloat();
        sum += z;

        System.out.format("The sum of number is : %f \n ",sum);

        input.close();
        
    }
}

File: C_LANGUAGE/viral4.c
#include<stdio.h>

void main()
{
    int n;
    printf("Enter the number :");
    scanf("%d",&n);

    int i;

    for(int i=1;i<=n;i++)
    {
        for (int j = 0; j < i ; j++)
        {               
            printf("*");
        }
        
       
        printf("\n");
    }
}